mne/stats/cluster_level.py

```diff
@@ -1190,11 +1204,21 @@ def permutation_cluster_1samp_test(
     to perform permutations. This might not be suitable for the case where
     there is truly a single observation under test; see :ref:`disc-stats`.
 
-    If ``n_permutations >= 2 ** (n_samples - (tail == 0))``,
+    For computing a ``threshold`` based on a p-value, use the conversion
+    from :meth:`scipy.stats.t.ppf`::
+
+        pval = 0.001  # arbitrary
+        df = n_observations - 1  # degrees of freedom for the test
+        thresh = scipy.stats.t.ppf(1 - pval /  2, df)  # two-tailed
```

Comment:
ah, here is the t test. But my previous remark still holds - you could be using t test to compare two groups instead of anova.